package.json

```diff
@@ -16,6 +22,13 @@
     "eslint-plugin-svelte3": "^3.2.0",
     "jest": "^27.2.5",
     "prettier": "^2.3.0",
+    "rollup": "^2.60.2",
+    "rollup-plugin-copy": "^3.4.0",
+    "rollup-plugin-css-only": "^3.1.0",
+    "rollup-plugin-livereload": "^2.0.5",
+    "rollup-plugin-serve": "^1.1.0",
+    "rollup-plugin-svelte": "^7.1.0",
```

Comment:
_*-livereload_, _*-serve_ and _*-svelte_ are only used in a single package each -> Not a good use case for centralising it in my book!

Just to be clear, I consider centralised dependencies to be _quite_ a hack: It adds complexity and slows down stuff like `./build.js lint` in exchange for saving a bit of time and disk space when doing a full build. Figuring out which of these are actually needed/used somewhere is pretty damn difficult, we have to look for `import`/`require` statements and hope that catches everything... I think we only want to pay that price for cases where it actually does speed up the build and save disk space. Dependencies only used by a single package are not an example of that IMO.